source/CIGameUpdate.h

```diff
@@ -135,13 +135,6 @@ class GameUpdate {
     int getTimestamp() {
         return _timestamp;
     }
-    
-    /**
-     * Returns whether the player sending the game update has won.
-     */
-    bool didPlayerWin() {
-        return _planet->isWinner();
-    }
```

Comment:
why did we get rid of this?

Reply:
oh it seemed like we weren't using it and was overlapping with the planetmodel's isWinner method. Were we planning on using it somewhere? If so I can add it back in.